README.md

```diff
@@ -1180,18 +1181,6 @@ Other Style Guides
     export { foo }
     ```
 
-  <a name="modules--prefer-default-export"></a>
-  - [10.6](#modules--prefer-default-export) In modules with a single export, prefer default export over named export.
- eslint: [`import/prefer-default-export`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md)
-
-    ```javascript
-    // bad
-    export function foo() {}
-
-    // good
-    export default function foo() {}
-    ```
-
```

Comment:
Wondering if, rather than removing this enty, we should edit it to something like...
```
10.6 In general, excluding React Component and Reducer files, named exports should be preferred.

// bad
...
export default function foo() {}

// good
...
export function foo() {} 

// also good
...
function foo() {}
export foo;

// also good
...
function foo() {}
export { foo };
```